starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -81,6 +84,20 @@ async def get_events(
     This is why we need to iterate block by block, take all events,
     and chunk it later which is not an optimal solution.
     """
+    # Required parameters
+    from_block = try_parse_from_filter(filter, "from_block")
+    to_block = try_parse_from_filter(filter, "to_block")
+    chunk_size = int(try_parse_from_filter(filter, "chunk_size"))
+
+    # Optional parameters
+    address = try_parse_from_filter(filter, "address")
+    keys = try_parse_from_filter(filter, "keys")
+    continuation_token = try_parse_from_filter(filter, "continuation_token")
+
+    # Set continuation_token to default value "0" when it's not set
+    if continuation_token is None:
```

Comment:
This could be
```python
filter.get("continuation_token", "0")
```

Reply:
Nice! I was looking exactly for something like this. Fixed in b34e920d6f114743464b8a6eb0f77db9c558a628